Documentation/development-flow.md

```diff
@@ -206,15 +206,19 @@ Rebasing is a very powerful feature of Git. You need to understand how it works
 
 ## Submitting a Pull Request
 
-Once you have implemented the feature or bug fix in your branch, you will open a PR to the upstream rook repo. Before opening the PR ensure you have added unit tests, are passing the integration tests, cleaned your commit history, and have rebased on the latest upstream.
+Once you have implemented the feature or bug fix in your branch, you will open a Pull Request (PR)
+to the [upstream Rook repository](https://github.com/rook/rook). Before opening the PR ensure you
+have added unit tests and all unit tests are passing. Please clean your commit history and rebase on
```

Comment:
Add an indication on how to run the unit test with `make`?

Reply:
Is a link to the section below good?